packages/@aws-cdk/aws-pipes-targets-alpha/lib/sagemaker.ts

```diff
@@ -0,0 +1,66 @@
+import { IInputTransformation, IPipe, ITarget, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import { IPipeline } from 'aws-cdk-lib/aws-sagemaker';
+
+/**
+ * SageMaker target properties.
+ */
+export interface SageMakerTargetParameters {
+  /**
+   * The input transformation to apply to the message before sending it to the target.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html#cfn-pipes-pipe-pipetargetparameters-inputtemplate
+   * @default - none
+   */
+  readonly inputTransformation?: IInputTransformation;
+
+  /**
+   * List of parameter names and values for SageMaker Model Building Pipeline execution.
+   *
+   * The Name/Value pairs are passed to start execution of the pipeline.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetsagemakerpipelineparameters.html#cfn-pipes-pipe-pipetargetsagemakerpipelineparameters-pipelineparameterlist
+   * @default - none
+   */
+  readonly pipelineParameters?: Record<string, string>;
+}
+
+/**
+ * A EventBridge Pipes target that sends messages to an SageMaker.
+ */
+export class SageMakerTarget implements ITarget {
+  private pipeline: IPipeline;
+  private sagemakerParameters?: SageMakerTargetParameters;
```

Comment:
nit: for consistency with the other classes and interfaces, can we make this param camel case as well?
```suggestion
  private sageMakerParameters?: SageMakerTargetParameters;
```